src/haddock/core/defaults.py

```diff
@@ -2,18 +2,26 @@
 import string
 import sys
 from pathlib import Path
+import os
 
 import yaml
 
 from haddock import core_path, haddock3_repository_path, log
 
 
 # Locate the CNS binary
-cns_exec = Path(haddock3_repository_path, "bin", "cns")
```

Comment:
FYI, the `cns_exec` is also a parameter in the configuration YAML files. Users can define the pah to `cns` as an alternative to the default `bin/cns/`.